docs/how-to/work-with-pinning-services.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Work with pinning services
+description: Learn how to use or create remote pinning services with IPFS, the InterPlanetary File System.
+---
+
+# Work with remote pinning services
+
+Depending on how you use IPFS, you might find it helpful to use a **remote pinning service** instead of, or in addition to, pinning files on your local IPFS node. Whether it happens remotely or locally, **pinning** an item in IPFS identifies it as something you always wish to keep available, exempting it from the routine "garbage collection" that IPFS does on infrequently-used items in order to efficiently manage storage space. (For more details on pinning as a concept, [see this guide](/how-to/pin-files).)
+
+If you've got just one local IPFS node, and it's always running, local pinning may be all you need to insure your important items are persisted and never garbage-collected. However, using a remote pinning service — or creating your own! — might be useful to you if:
+
+- Your local node isn't always online, but you need items to be consistently available
+- You'd like to keep a persistent backup of your local node's files somewhere else
+- You don't have all the disk space you need on your local node
+- You run more than one IPFS node, and would like to use one of them as a "personal pinning service" as your preferred location for permanent storage
+
+There are a number of commercial pinning services that make it easy for you to purchase pinning capacity for your important files (examples of these include Pinata, Temporal, Infura, and others). Each of these third-party services has its own unique interface for pinning files and managing those pins; this could include a GUI, an API, CLI commands, or other tooling.
```

Comment:
```suggestion
There are a number of commercial pinning services that make it easy for you to purchase pinning capacity for your important files, some of which include Pinata, Temporal, Infura, and others. Each of these third-party services has its own unique interface for pinning files and managing those pins; this could include a GUI, an API, CLI commands, or other tooling.
```